android/app/build.gradle

```diff
@@ -51,7 +51,7 @@ android {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "co.nimblehq.flutter.template"
         minSdkVersion 16
-        targetSdkVersion 30
+        targetSdkVersion 31
```

Comment:
actually we shouldn't include these things here, but if you did then please add these details to your PR so later we can ref to

Reply:
@sleepylee updated